tests/migrations/models.py

```diff
@@ -72,3 +72,11 @@ class FoodManager(BaseFoodManager.from_queryset(FoodQuerySet)):
 
 class NoMigrationFoodManager(BaseFoodManager.from_queryset(FoodQuerySet)):
     pass
+
+
+class ExoPlanet(models.Model):
+    name = models.CharField(max_length=255)
+    inhabitable = models.BooleanField(default=False)
+
+    class Meta:
+        app_label = 'exoplanets'
```

Comment:
I think we need a comment explaining this explicit `app_label`. @charettes, can you say if this will this cause any problems with app registry consistency?